app/models/user.rb

```diff
@@ -230,6 +228,10 @@ def ants_pre_demande_number=(value)
     super(value&.upcase)
   end
 
+  def timeout_in
```

Comment:
ça serait chouette d'avoir le commentaire d'explication ici aussi (sinon ça ressemble beaucoup à du code mort)
```suggestion
  def timeout_in  # Used by Devise's :timeoutable
```

(on aurait aussi presque pu mettre la méthode en one-liner juste en dessous de l'appel à devise pour mieux grouper les choses)